app/uk/gov/hmrc/soletraderidentificationfrontend/views/personal_information_error_page.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.soletraderidentificationfrontend.views.html.helpers._
+@import uk.gov.hmrc.soletraderidentificationfrontend.views.helpers.TitleHelper.title
+
+@this(layout: govuk_template,
+govukFieldset: GovukFieldset,
+govukButton: GovukButton,
+formWithCSRF: FormWithCSRF,
+inputDate: inputDate,
+errorSummary: errorSummary
+)
+
+@(formAction: Call)(implicit request: Request[_], messages: Messages)
+
+@layout(title = messages("personal-information-error.title")) {
+
+    @govukFieldset(Fieldset(
+        legend = Some(Legend(
+            content = Text(Messages("personal-information-error.heading")),
+            classes = "govuk-fieldset__legend--xl",
+            isPageHeading = true
+        )),
+        html = html
+        ))
+    }
+
+@html = {
+    @formWithCSRF(action = formAction) {
+        <p class="govuk-body">@messages("personal-information-error.line_1")</p>
+
+        @govukButton(Button(
+            attributes = Map("id" -> "try-again"),
+            classes = "govuk-!-margin-right-1",
```

Comment:
you don't need `classes`. you need `href` to add the link for the start of the journey